integration_test/testutil_test.go

```diff
@@ -291,16 +293,29 @@ func (it *ITest) initializeIndex() {
 		}
 	})
 
+	// check for KREW_DEFAULT_INDEX_URI
+	for _, e := range it.env {
+		if strings.Contains(e, "KREW_DEFAULT_INDEX_URI") {
+			val := strings.Split(e, "=")[1]
+			it.extractKrewIndexTar(val)
+			it.Krew("index", "add", "default", val).RunOrFail()
+			return
+		}
+	}
```

Comment:
Let's not complicate the test setup further. This adds considerable complexity for a one-line config change.

I'd prefer not to have a special case for `KREW_DEFAULT_INDEX_URI` here.